arch/x86_64/core/xuk-stub32.ld

```diff
@@ -20,4 +20,24 @@ SECTIONS {
 
 	. = ALIGN(16);
 	_start64 = .;
+
+	/* As of an update near 1 May 2019, Ubuntu's 18.04 toolchain
```

Comment:
Please mention the exact package and version change considering we found it/narrowed it down:
 binutils  2.30-21ubuntu1~18.04 -> 2.30-21ubuntu1~18.04.1

Reply:
I tried both versions and it made no difference.